### DISCUSSION TITLE
Garrysmod icon as a gamemode

### DISCUSSION THREAD

**Discussion by Stinggy** [tltneon/lgsl#84] [Ideas]
Curious if you could add a drop down for garry's mod in addition to half life and add a icon for Garry's mod.

Great work on the project!

Thanks

**Comment by tltneon** [comment]
Icon for Garry's Mod already added to LGSL if I got your point
https://github.com/tltneon/lgsl/blob/master/lgsl_files/icons/source/garrysmod.gif
